workers/api-server/src/router.rs

```diff
@@ -35,25 +35,41 @@ const ERR_INVALID_QUERY_PARAMS: &str = "invalid query params";
 
 /// Builds an empty HTTP 400 (Bad Request) response
 pub(super) fn build_400_response(err: String) -> Response<Body> {
-    Response::builder().status(StatusCode::BAD_REQUEST).body(Body::from(err)).unwrap()
+    Response::builder()
+        .status(StatusCode::BAD_REQUEST)
+        .header("Access-Control-Allow-Origin", "*")
+        .body(Body::from(err))
+        .unwrap()
```

Comment:
nit: I suggest refactoring this into a helper like `build_err_resp_with_status(status: StatusCode, err: String)` given that all these bodies do the same thing with different status codes

Reply:
Replaced helper functions with already existing `build_response_from_status_code` fn